### DISCUSSION TITLE
Move to an official Rollup plugin?

### DISCUSSION THREAD

**Discussion by jonkoops** [thgh/rollup-plugin-livereload#87] [General]
As mentioned in https://github.com/rollup/plugins/issues/383, there is interest in making this project an official Rollup plugin. Let's use this topic to discuss any steps that we might have to take.

**Comment by jonkoops** [comment]
@thgh I think I can start off by creating an initial PR for putting what we have in `main` now in an official plugin (plus some refactoring to make it fit), WDYT?

**Reply by thgh** [reply]
I want the best for the Rollup ecosystem but won't take extra responsibilities. So once it's in the @rollup/plugins repo, it's all in your hands 🙂  (and shellscape)
I will take responsibility for this repo.
I will put a message in the README to let people know that @rollup/plugins-livereload exists (when it does).

**Reply by jonkoops** [reply]
Very well, then I will start the process of creating a PR towards the Rollup plugins repo :+1:
